src/helper/number.js

```diff
@@ -12,6 +12,11 @@ export function toNumber(number) {
 
   } else if (typeof number === 'string') {
     result = number.indexOf('.') > -1 ? parseFloat(number) : parseInt(number, 10);
+  } else if (number instanceof Date) {
+    // Excel serial dates are expressed as number of days since January 1, 1900
+    const unix = number.getTime() / 1000;
+    const excel = (unix / 86400) + 25569;
+    result = excel;
   }
 
```

Comment:
We will have an issue when the date in the string representation
parser.parser.yy.toNumber("2019-10-01T00:00:00.000Z")
2019
parser.parser.yy.toNumber(new Date())
43767.65573221065